src/examples/PrintExamplePage.tsx

```diff
@@ -0,0 +1,45 @@
+'use strict';
+import {Button, Linking, Text, View} from 'react-native';
+import React from 'react';
+import {Example} from '../components/Example';
+import {Page} from '../components/Page';
+import RNPrint from 'react-native-print';
+
+export const PrintExamplePage: React.FunctionComponent<{}> = () => {
+  const example = `import RNPrint from 'react-native-print';
+
+function Example() {
+  const printRemotePDF = async() => {
+    await RNPrint.print({ filePath: 'https://www.w3.org/WAI/ER/tests/xhtml/testfiles/resources/pdf/dummy.pdf' })
+  }
+
+  return (
+    <Button onPress={printRemotePDF} title="Print PDF" />
+  );
+}`;
+
+  const printRemotePDF = async() => {
+    await RNPrint.print({ filePath: 'https://www.w3.org/WAI/ER/tests/xhtml/testfiles/resources/pdf/dummy.pdf' })
+  }
+
+  return (
+    <Page
+      title="Print"
+      description="Print documents via the react-native-print module.">
+
+      <View >
+        <Text>For more information about the supported APIs, check out the{' '}
+          <Text style={{color: 'blue'}} onPress={() =>
+             Linking.openURL('https://github.com/christopherdro/react-native-print#usage')}>
+          documentation
+          </Text>
+        .
+        </Text>
+      </View>
```

Comment:
PR just merged which adds a hyperlink section at the bottom of each component page with links to documentation. See the template page as an example. Could you add that section to this page in replacement of this?